### ISSUE TITLE
Quest passthrough no longer working in StereoKitTest 

### ISSUE THREAD

**Issue by jackdaus** [StereoKit/StereoKit#594]
## Description
It looks like something might have changed in a recent SK preview version (v3.7-preview.7) that broke the StereoKitTest passthrough demo! When I try to run the StereoKitTest project on my Quest 2, the passthrough no longer works as expected. Where I expect to see passthrough video, I see black pixels.

I noticed this when I was updating one of my projects to the latest SK preview, but the passthrough wasn't working. I did some sleuthing, and it looks like the breaking change was commit bf0869c092a6c11b0641d6cce3c0ea2e4c71dfa2. Before this commit, the passthrough was working in the StereoKitTest project. 

## Platform / Environment
- Windows 11
- Meta Quest 2
- Quest Link
- SK version: a commit between 3.7-preview.6 and 3.7-preview.7

## Logs or exception details
Nothing super interesting in the logs here, but including them just in case:
```
[SK diagnostic] Initializing StereoKit v0.3.7-preview.6 Win32 x64...
[SK diagnostic] Initializing Platform
[SK diagnostic] sk_gpu: Requesting debug Direct3D context.
[SK diagnostic] sk_gpu: Using Direct3D 11: NVIDIA GeForce GTX 1650 Ti with Max-Q Design
[SK diagnostic] Starting mixed reality mode
[SK diagnostic] available: XR_KHR_D3D12_enable
[SK diagnostic] available: XR_KHR_opengl_enable
[SK diagnostic] available: XR_KHR_vulkan_enable
[SK diagnostic] available: XR_KHR_vulkan_enable2
[SK diagnostic] available: XR_KHR_convert_timespec_time
[SK diagnostic] available: XR_KHR_composition_layer_cube
[SK diagnostic] available: XR_KHR_composition_layer_cylinder
[SK diagnostic] available: XR_KHR_composition_layer_equirect
[SK diagnostic] available: XR_KHR_visibility_mask
[SK diagnostic] available: XR_KHR_composition_layer_color_scale_bias
[SK diagnostic] available: XR_EXT_win32_appcontainer_compatible
[SK diagnostic] available: XR_EXT_debug_utils
[SK diagnostic] available: XR_OCULUS_recenter_event
[SK diagnostic] available: XR_FB_display_refresh_rate
[SK diagnostic] available: XR_META_performance_metrics
[SK diagnostic] available: XR_OCULUS_ovrsession_handle
[SK diagnostic] available: XR_FB_hand_tracking_aim
[SK diagnostic] available: XR_FB_hand_tracking_capsules
[SK diagnostic] available: XR_FB_hand_tracking_mesh
[SK diagnostic] available: XR_FB_body_tracking
[SK diagnostic] available: XR_FB_eye_tracking_social
[SK diagnostic] available: XR_FB_face_tracking
[SK diagnostic] available: XR_FB_keyboard_tracking
[SK diagnostic] available: XR_FB_triangle_mesh
[SK diagnostic] available: XR_FB_render_model
[SK diagnostic] available: XR_FBX1_plane
[SK diagnostic] available: XR_FB_spatial_entity_container
[SK diagnostic] available: XR_FB_scene
[SK diagnostic] available: XR_FB_spatial_entity
[SK diagnostic] available: XR_FB_spatial_entity_storage
[SK diagnostic] available: XR_FB_spatial_entity_query
[SK diagnostic] available: XR_FB_spatial_entity_user
[SK diagnostic] available: XR_FB_spatial_entity_storage_batch
[SK diagnostic] available: XR_FB_spatial_entity_sharing
[SK diagnostic] available: XR_FB_haptic_amplitude_envelope
[SK diagnostic] available: XR_FB_haptic_pcm
[SK diagnostic] available: XR_FB_touch_controller_pro
[SK diagnostic] available: XR_FB_touch_controller_proximity
[SK diagnostic] available: XR_OCULUS_external_camera
[SK diagnostic] available: XR_FBX1_touch_controller_extras
[SK diagnostic] REQUESTED: XR_KHR_D3D11_enable
[SK diagnostic] REQUESTED: XR_KHR_composition_layer_depth
[SK diagnostic] REQUESTED: XR_KHR_win32_convert_performance_counter_time
[SK diagnostic] REQUESTED: XR_OCULUS_audio_device_guid
[SK diagnostic] REQUESTED: XR_FB_color_space
[SK diagnostic] REQUESTED: XR_EXT_hand_tracking
[SK diagnostic] REQUESTED: XR_FB_passthrough
[SK diagnostic] REQUESTED: XR_EXT_palm_pose
[SK diagnostic] Using system: Oculus Quest2
[SK diagnostic] Platform supports single-pass rendering
[SK diagnostic] OpenXR articulated hands ext enabled!
[SK diagnostic] OpenXR depth LSR ext enabled!
[SK diagnostic] Set color space to XR_COLOR_SPACE_REC709_FB
[SK diagnostic] Creating view: PrimaryStereo color:rgba32_sRGB depth:depth32 blend:Opaque
[SK diagnostic] Setting view: PrimaryStereo to 1568x1568
[SK diagnostic] Bounds updated: 3.82x5.55 at (0.0,-1.2,0.0) (0.00,0.00,0.00,1.00)
[SK diagnostic] Initializing Audio
[SK info] Using audio backend: WASAPI
[SK diagnostic] Initializing Assets
[SK diagnostic] Initializing Defaults
[SK diagnostic] Initializing World
[SK diagnostic] Initializing Sprites
[SK diagnostic] Initializing Lines
[SK diagnostic] Initializing UI
[SK diagnostic] Initializing Renderer
[SK diagnostic] Initializing Physics
[SK diagnostic] Initializing Input
[SK info] Initialization successful
[SK info] XrTime: 7573180917
[SK diagnostic] sprite_create: Atlased sprites not implemented yet! Switching to single.
[SK info] Starting Scene: DemoWelcome
[SK diagnostic] OpenXR session begin.
[SK info] App focus changed to: Active
[SK info] Starting Scene: DemoFBPassthrough
[SK info] xrEndFrame [XR_ERROR_HANDLE_INVALID]
[SK info] Bye!
[SK info] Session Performance Report:
[SK info] ______________________________________________________
[SK info] |         System | Initialize |   Update |  Shutdown |
[SK info] |________________|____________|__________|___________|
[SK info] |       Defaults |    78.80ms |          |    0.19ms |
[SK info] |     FrameBegin |            |  0.104ms |           |
[SK info] |          Input |    11.78ms |  0.053ms |    0.06ms |
[SK info] |        Physics |     3.34ms |  0.066ms |    0.05ms |
[SK info] |       Renderer |     2.94ms |  0.000ms |    0.20ms |
[SK info] |             UI |     6.09ms |  0.012ms |    0.28ms |
[SK info] |       Platform |   819.67ms |          |  103.75ms |
[SK info] |            App |   134.05ms |  1.221ms |           |
[SK info] |      Animation |            |  0.000ms |    0.00ms |
[SK info] |          Lines |     0.77ms |  0.019ms |    0.01ms |
[SK info] |        Sprites |     1.26ms |  0.000ms |    0.17ms |
[SK info] |           Text |            |  0.020ms |    0.01ms |
[SK info] |         UILate |            |  0.000ms |           |
[SK info] |          World |     1.02ms |  0.001ms |    0.01ms |
[SK info] |    FrameRender |            |  0.827ms |           |
[SK info] |         Assets |     3.35ms |  0.002ms |    0.01ms |
[SK info] |          Audio |   176.97ms |  0.004ms |    4.67ms |
[SK info] |________________|____________|__________|___________|

C:\Git\StereoKit\StereoKit\bin\x64_Debug\StereoKitTest\StereoKitTest.exe (process 24676) exited with code 0.
To automatically close the console when debugging stops, enable Tools->Options->Debugging->Automatically close the console when debugging stops.
Press any key to close this window . . .
```

**Comment by jackdaus** [comment]
It looks like reverting commit bf0869c092a6c11b0641d6cce3c0ea2e4c71dfa2 on the latest develop _does_ fix the passthrough issue. I can go ahead and make a PR of that change if that's at all helpful? 

Swapchain formats are a bit beyond my depth at this point! So I'm not sure if doing a simple revert here is the best fix for this particular issue. But at least it's an option to get the FB passthrough working again, short term.

**Comment by maluoi** [comment]
Thank you for pinning this down to the exact commit! I had a bit of a local Android build blocker that I _just_ found my way through, so I'll be investigating this a bit deeper pretty soon :)

I was so excited about getting rid of that hack 😩 It looked like it behaved fine, but I guess it was a little optimistic! Also interesting that this still seems to work fine on standalone Quest.

**Comment by maluoi** [comment]
This should be fixed in 0.3.7-preview.1570!
